integration-cli/docker_cli_pause_test.go

```diff
@@ -0,0 +1,95 @@
+package main
+
+import (
+	"fmt"
+	"os/exec"
+	"strings"
+	"testing"
+	"time"
+)
+
+func TestEventsPause(t *testing.T) {
+	name := "testeventpause"
+	out, _, _ := dockerCmd(t, "images", "-q")
+	image := strings.Split(out, "\n")[0]
+	dockerCmd(t, "run", "-d", "--name", name, image, "sleep", "2")
+	dockerCmd(t, "pause", name)
+	dockerCmd(t, "unpause", name)
+
+	defer deleteAllContainers()
```

Comment:
Can move this defer to the top then if things die during the pause/unpause we'll still clean-up?